test/functional/plugin/shada_spec.lua

```diff
@@ -2554,6 +2548,8 @@ describe('syntax/shada.vim', function()
   it('works', function()
     nvim_command('syntax on')
     nvim_command('setlocal syntax=shada')
+    -- TODO(justinmk): why is this needed?
+    nvim_command('set laststatus&')
```

Comment:
Still wondering why this is needed myself, and how you came up with it?
(https://github.com/neovim/neovim/pull/10701#discussion_r311005754)

Reply:
I just compared `helpers.nvim_args`.

Reply:
Ah.  Should I remove the TODO here then?
Might be good to have an explanation why it is needed (if you happen to have an idea).

Reply:
it's just needed to make the test pass. Don't know why, almost certainly just how shada works. Could adjust the test somehow bit didn't want to spend time on it.